validation/open_boundaries/timevarying_input.jl

```diff
@@ -0,0 +1,50 @@
+using Oceananigans
+
+using Oceananigans.BoundaryConditions: OrlanskiBoundary
+using Oceananigans.TimeSteppers: QuasiAdamsBashforth2TimeStepper
+
+grid = RectilinearGrid(topology = (Bounded, Bounded, Bounded), size = (128, 16, 16), extent = (40, 5, 5))
+
+timestepper = QuasiAdamsBashforth2TimeStepper(grid, ())
+
+u_orlanski = OpenBoundaryCondition(OrlanskiBoundary(timestepper.Gⁿ.u))
```

Comment:
Why do we have to pass the tendency here? Isn't there a wave speed that can be set?

Reply:
I don't think so, it seems like the wave speed is usually estimated as $-\frac{\partial_t\phi}{\partial_x\phi}$ where $\phi$ is whatever is going through the boundary